skrub/_expressions/_parallel_coord.py

```diff
@@ -90,14 +100,21 @@ def _prepare_obj_column(col):
     }
 
 
-def _prepare_numeric_column(col, log_scale):
+def _prepare_numeric_column(col, log_scale, is_int):
     vals = col.to_numpy()
     if log_scale:
         vals = np.log(vals)
     min_val, max_val = np.nanmin(vals), np.nanmax(vals)
     tickvals = np.linspace(min_val, max_val, 10).tolist()
-    if pd.api.types.is_integer_dtype(col):
-        ticktext = [str(int(np.round(np.exp(v)))) for v in tickvals]
+    if pd.api.types.is_integer_dtype(col) or is_int:
+        if log_scale:
+            tickvals = np.exp(tickvals)
+        tickvals = np.unique(np.round(tickvals).astype("int"))
+        tickvals_label_space = tickvals.tolist()
+        if log_scale:
+            tickvals = np.log(tickvals)
+        tickvals = tickvals.tolist()
+        ticktext = list(map(str, tickvals_label_space))
     else:
         ticktext = list(
             map("{:.2g}".format, np.exp(tickvals) if log_scale else tickvals)
```

Comment:
Instead of the `map` and because we call list, I would just do a list comprehension:

```python
ticktext = [f"{np.exp(val) if log_scale else val:.2g}" for val in tickvals]
```